sass/components/_menu.scss

```diff
@@ -0,0 +1,37 @@
+
+.menu {
+    
+    list-style-type: none;
+    padding: 20px 40px;
+    @include breakpoint($breakpoint-small) {
+        padding: 40px 20px;
+        font-size: 2em;
+        $menu-width: 100%;
+        position: absolute;
+        text-align: center;
+        width: $menu-width;
+        background: #3dd8c8;
+        top: -$menu-width;
```

Comment:
Tutaj na pewno `$menu-width`? Bardziej bym obstawiał że coś z `height` ale ty lepiej to sprawdzisz